src/style/fims/_base.scss

```diff
@@ -170,4 +170,58 @@ md-checkbox.small {
 .form-horizontal .form-group {
     margin-left: 0px;
     margin-right: 0px;
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+
+//about page/splashpage
```

Comment:
can we format the code that follows. Also, I'm trying to keep any css in this file for stuff that is used in many places.  Any specific "views" should have their own sass file and just import it in "_fims.scss" (ex. "_dashboard.scss"). If the sass if for a single component then it can be imported directly in the component (ex. "_projectSelector.scss" and "projectSelectorDialog.component.js".